src/hooks/use-canvas.ts

```
@@ -0,0 +1,208 @@
+            case 'frame':
+            case 'rect':
+            case 'ellipse':
+            case 'generatedui':
+                return (
+                    point.x >= shape.x &&
+                    point.x <= shape.x + shape.w &&
+                    point.y >= shape.y &&
+                    point.y <= shape.y + shape.h
+                )
+            case 'freedraw':
+                const threshold = 5
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)